general/clients/web-config.md

```diff
@@ -0,0 +1,52 @@
+---
+uid: web-config
+title: Jellyfin Web configuration
+---
+
+# Jellyfin Web configuration
+
+## Editing the configuration
+
+The Jellyfin Web default interface can be configured using the `config.json` file in the webroot. Where this is and how to edit it depends on the installation method.
```

Comment:
So let me ask this. If I want to disable the youtube plugin to be more private. Can't I bypass this by using a web client in standalone mode?

Reply:
yes